lib/helpers/hearing_renderer.rb

```diff
@@ -0,0 +1,358 @@
+# frozen_string_literal: true
+
+class HearingRenderer
+  RENDERABLE_CLASSNAMES = %w[
+    Veteran
+    Appeal
+    LegacyAppeal
+    Hearing
+    LegacyHearing
+    HearingTask
+    VirtualHearing
+  ].freeze
+
+  class << self
+    def test_veteran_with_ama
+      ama_virtual_hearing = VirtualHearing.not_cancelled.where(hearing_type: Hearing.name).last
+      veteran_with_ama_hearing = ama_virtual_hearing.hearing.appeal.veteran
+      puts render(veteran_with_ama_hearing)
+    end
+
+    def test_veteran_with_ama_with_pii
+      ama_virtual_hearing = VirtualHearing.not_cancelled.where(hearing_type: Hearing.name).last
+      veteran_with_ama_hearing = ama_virtual_hearing.hearing.appeal.veteran
+      puts render(veteran_with_ama_hearing, show_pii: true)
+    end
+
+    def test_veteran_with_legacy
+      legacy_virtual_hearing = VirtualHearing.not_cancelled.where(hearing_type: LegacyHearing.name).last
+      veteran_with_legacy_hearing = legacy_virtual_hearing.hearing.appeal.veteran
+      puts render(veteran_with_legacy_hearing)
+    end
+
+    def test_veteran_with_legacy_with_pii
+      legacy_virtual_hearing = VirtualHearing.not_cancelled.where(hearing_type: LegacyHearing.name).last
+      veteran_with_legacy_hearing = legacy_virtual_hearing.hearing.appeal.veteran
+      puts render(veteran_with_legacy_hearing, show_pii: true)
+    end
+
+    def test_breadcrumbs
+      ama_virtual_hearing = VirtualHearing.not_cancelled.where(hearing_type: Hearing.name).last
+      puts render(ama_virtual_hearing.hearing)
+    end
+
+    def render(obj, show_pii: false)
+      renderer = HearingRenderer.new(show_pii: show_pii)
+      TTY::Tree.new(renderer.structure(obj, include_breadcrumbs: true)).render
+    end
+
+    def renderable_classes
+      RENDERABLE_CLASSNAMES.map(&:constantize)
+    end
+
+    def prefix(obj)
+      klass = renderable_classes.find { |k| obj.is_a?(k) }
```

Comment:
suggesting codeclimate fix
```suggestion
      klass = renderable_classes.find { |klass| obj.is_a?(klass) }
```